altair/vegalite/v2/examples/scatter_linked_brush.py

```diff
@@ -0,0 +1,25 @@
+"""
+Faceted Scatter Plot with Linked Brushing
+-----------------------------------------
+This is an example of using an interval selection to control the color of
+points across multiple facets.
+"""
+
+import altair as alt
+from vega_datasets import data
+
+cars = data.cars()
+
+brush = alt.selection(type='interval', resolve='global')
+
+base = alt.Chart(cars).mark_point().encode(
+    y='Miles_per_Gallon',
+    color=brush.condition('Origin',
+                          alt.ColorValue('gray'))
```

Comment:
This is not the point of this PR, but I think I should ask anyway.

I wonder why this has to be `alt.ColorValue('gray')` -- wouldn't `alt.Value('gray')` or something similar sufficient?  

(At the end of the day, it's just `{value: "gray"}` in JSON.)

Reply:
The way the schema is set up, the encodings all have slightly different definitions (i.e. ``color`` is a ``MarkPropValueDefWithCondition``, ``text`` is a ``TextValueDefWithCondition``, ``x`` is a ``ValueDef``, etc.) Given that every channel has a specific value type that it can accept, I thought the easiest approach for usability would be to alias the particular value type to the channel name.

That way, the user doesn't need a lookup table to remember that ``x`` should accept a ``ValueDef``. They just need to remember that ``x`` takes an ``XValue()``.